doc/ref_model/chapters/glossary.md

```diff
@@ -73,7 +74,23 @@ To help guide the reader, this glossary provides an introduction to the terminol
 - **Service Assurance (SA)**: collects alarm and monitoring data. Applications within SA or interfacing with SA can then use this data for fault correlation, root cause analysis, service impact analysis, SLA management, security, monitoring and analytic, etc.
 
 <a name="11.1.4"></a>
-### 1.1.4 Other Referenced Terminology
+### 11.1.4 Container Related Terminology
+
+>_*Note:*_ Relevant terms are added here from RA2. Most of these term definitions are taken from [kubernetes glossary](https://kubernetes.io/docs/reference/glossary) but in some cases should be made independent from kubernetes as a specific container orchestration engine.
```

Comment:
Should capitalise Kubernetes (i.e. from kubernetes).

Reply:
ok done